excel/excelFileReader.js

```diff
@@ -26,7 +26,8 @@ module.exports = class ExcelFileReader {
         return text.trim()
             .replace(/\'|\u2018|\u2019|\u201b|\u2032/g, "\\\'")
             .replace(/\"|\u201c|\u201d|\u2033/g, "\\\"")
-            .replace(/\r\n/g, "\\n");
+            .replace(/\r\n/g, "\\n")
+            .replace(/&\s/g, "&amp; ");
```

Comment:
is the trailing space in `"&amp; " supposed to exist?

Reply:
To avoid making &amp; become &amp;amp;

Reply:
Your comment made me aware of other potential issue. See new code please.